yarn.lock

```diff
@@ -4,8 +4,8 @@
 
 "@babel/cli@7.8.4":
   version "7.8.4"
-  resolved "https://registry.yarnpkg.com/@babel/cli/-/cli-7.8.4.tgz#505fb053721a98777b2b175323ea4f090b7d3c1c"
-  integrity sha512-XXLgAm6LBbaNxaGhMAznXXaxtCWfuv6PIDJ9Alsy9JYTOh+j2jJz+L/162kkfU1j/pTSxK1xGmlwI4pdIMkoag==
+  resolved "http://packages.je-labs.com/npm/private-npm/@babel/cli/-/cli-7.8.4.tgz#505fb053721a98777b2b175323ea4f090b7d3c1c"
```

Comment:
`packages.je-labs.com` references in the lockfile are causing travis to complain.

Reply:
@benwhite-deltas fixed.